app/views/ems_infra/_show_dashboard.html.haml

```diff
@@ -1,108 +1,16 @@
-                   "donut-config"               => "cpuUsageDonutConfig",
-                   "pf-utilization-trend-chart" => "",
-                   "sparkline-config"           => "cpuUsageSparklineConfig"}
-            %span.trend-footer-pf{"ng-class" => "{ 'chart-transparent-text': !cpuUsageData }"}
-              = _("Last 30 Days")
-
-          .col-xs-6.col-sm-6.col-md-6
-            %div{"ng-if" => "memoryUsageData"}
-              %div{"chart-data"                 => "memoryUsageData",
-                   :config                      => "memoryUsageConfig",
-                   "donut-config"               => "memoryUsageDonutConfig",
-                   "pf-utilization-trend-chart" => "",
-                   "sparkline-config"           => "memoryUsageSparklineConfig"}
-            %span.trend-footer-pf{"ng-class" => "{ 'chart-transparent-text': !memoryUsageData }"}
-              = _("Last 30 Days")
-
+      = render :partial => "utilization-trend-chart"
     .col-xs-12.col-sm-6.col-md-6
-      %div{"column-sizing-class"          => "col-xs-8 col-sm-6 col-md-6",
-           "heat-map-usage-legend-labels" => "nodeHeatMapUsageLegendLabels",
-           "heatmap-chart-height"         => "dashboardHeatmapChartHeight",
-           :heatmaps                      => "heatmaps",
-           "heatmaps-card"                => "",
-           :hidetopborder                 => "true",
-           "title"                        => "{{clusterChartTitle}}"}
-  .row.row-tile-pf.row-tile-pf-last
+      = render :partial => "heatmap"
+  .row.row-tile-pf
     .col-xs-12.col-sm-6.col-md-6
-      %div{"head-title" => "{{recentHostsConfig.headTitle}}",
-           "pf-card"    => ""}
-
-        .spinner.spinner-lg.loading{"ng-if" => "!loadingDone"}
-        %div{"chart-data"      => "recentHostsData",
-             "chart-height"    => "chartHeight",
-             :config           => "recentHostsConfig",
-             "ng-if"           => "loadingDone",
-             "pf-line-chart" => ""}
-        %span.trend-footer-pf{"ng-class" => "{ 'chart-transparent-text': !recentHostsConfig }"}
-          = _("Last 30 Days")
-
+      = render :partial => "recent-hosts-line-chart"
     .col-xs-12.col-sm-6.col-md-6
-      %div{"head-title" => "{{recentVmsConfig.headTitle}}",
-           "pf-card"    => ""}
-
-        .spinner.spinner-lg.loading{"ng-if" => "!loadingDone"}
-        %div{"chart-data"      => "recentVmsData",
-             "chart-height"    => "chartHeight",
-             :config           => "recentVmsConfig",
-             "ng-if"           => "loadingDone",
-             "pf-line-chart" => ""}
-        %span.trend-footer-pf{"ng-class" => "{ 'chart-transparent-text': !recentVmsConfig }"}
-          = _("Last 30 Days")
+      = render :partial => "recent-vms-line-chart"
 
 :javascript
-  miq_bootstrap('.ems-infra-dashboard');
+  ManageIQ.angular.app.value('providerId', '#{@record.id}');
+  miq_bootstrap('.ems-infra-dashboard', '');
```

Comment:
Without that second argument please :) Or if you want to be bootstrapping a different module, don't name it an empty string.